src/libs/NextStepUtils.ts

```diff
@@ -121,7 +116,14 @@ function buildNextStep(report: OnyxEntry<Report>, predictedNextStatus: ValueOf<t
             if (harvesting?.enabled && autoReportingFrequency !== CONST.POLICY.AUTO_REPORTING_FREQUENCIES.MANUAL) {
                 optimisticNextStep.message = [
                     {
-                        text: 'These expenses are scheduled to ',
+                        text: 'Waiting for ',
```

Comment:
I don't have access to [Update Report Next Steps](https://docs.google.com/document/d/1mUUQzep18IWdZpwG6Qqtw9JH0dvLMvLXh-KTn_B51I4/edit#heading=h.mnjguil21u0a) design doc, correct me if I'm wrong, but I think this should say
"Waiting for ownerDisplayName`'s` expense(s) to automatically submit"

<img width="279" alt="Screenshot 2024-07-18 at 12 13 41" src="https://github.com/user-attachments/assets/5a0ade1f-8b3e-4b4c-8dfd-c2cdd5fb4144">
 So instead of this, it would say "Waiting for Rayane's expense(s) to automatically submit..."
 What do you think?

Reply:
https://expensify.slack.com/archives/C02NK2DQWUX/p1721299079830159